### ISSUE TITLE
Māori not included in version 1.50?

### ISSUE THREAD

**Issue by martino-vic** [espeak-ng/espeak-ng#1001]
Hi, I want to transcribe Māori to IPA. I'm using espeak-ng 1.50 through phonemizer 2.2.2 on Windows 10 and Python 3.9. I'm running following code:
```
>>> import os
>>> from phonemizer import phonemize

>>> os.environ["PHONEMIZER_ESPEAK_PATH"] = r"C:\Program Files (x86)\eSpeak NG\espeak-ng.exe"
>>> phonemize("lol", backend="espeak", language="mi")
RuntimeError: language "mi" is not supported by the espeak backend
```

It's similar to this issue: https://github.com/espeak-ng/espeak-ng/issues/827  
If I run ``` espeak-ng --voices ``` in my shell, ```mi``` is not included in the list. However, at https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages.md it *is* inlcuded. Why is there a difference between the two lists and should it be possible to transcribe Māori or not?  
Update: Seems like the docs are referring to the development version which is different from the distributed version. But still: Māori was included in at least one earlier version (1.49.2) - why would it have been taken out? And is it possible to download the development version somehow, instead of the last stable release (1.50) from 2019, if that would be the solution? 
Ps. the contribution guidelines tell me to test this issue on eSpeak NG online first, but as it turns out, that site is based on the development version, not the stable release, so the error is not reproducible there.

**Comment by martino-vic** [comment]
Based on this issue: https://github.com/espeak-ng/espeak-ng/issues/469 Māori was supposed to be included in version 1.49.2 (in 2017) according to release notes but in the end wasn't. It was only part of the development version (2018) and apparently hasn't made it into the stable version 1.50 (Dec. 2019) either?

**Comment by jaacoppi** [comment]
Māori support has been included since 1.49.2. 

However, we currently have a problem with the Windows installer missing some languages. To get Māori support for Windows, either: 
1) compile from source: https://github.com/espeak-ng/espeak-ng/blob/master/docs/building.md
2) help us improve the Windows installer: See #891.

**Comment by martino-vic** [comment]
Ah great to know! Thank you so much!
